active/ai.md

```diff
@@ -0,0 +1,97 @@
+# AI Team
+
+The Artificial Intelligence Team encourages thoughtful adoption of Artificial Intelligence by Django users and developers.
+
+## Scope of responsibilities
+
+The scope of the AI Team includes:
+
+- Making it easier to build AI/ML products with Django without compromising on ethical, safety, social responsibility fundamentals
```

Comment:
I have concerns around the phrase "Making it easier to build AI/ML products with Django". Using this phrase in the charter of a team implies that changes to Django are within the scope of the team.

Reply:
I wonder if it could be rephrased to remove that implication? E.g. I think the AI team could serve this goal via documentation, tutorials, etc. without having to make any changes to DJango.